### PR TITLE
TabbedCard component

### PR DESCRIPTION
**Pull request by hooloovooo**
## Hey, I just made a Pull Request!

TabbedCard component

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [x] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

2020-04-07-103726_1019x165_scrot

### CODE REVIEW

**Comment by hooloovooo** [comment]
Veery early example. I'm creating PR to discuss how we want to do with controlled/uncontrolled mode of the component. Currently it's just a very thin layer on top of Material UI.
Do we want it to work without manually setting the values of the selected tab etc, and if we do, do we still want to keep the option of setting it manually?
I've also moved the BottomLink component to the layout folder for reusing it in TabbedCard. It feels a bit wrong to have it there, but i can't think of a better place atm. Any suggestions?

**Comment by hooloovooo** [comment]
For some reason the PR seems a bit broken. I based it on the InfoCard typescript branch and now GitHub doesn't realise that has already been merged...

**Comment by stefanalund** [comment]
Could you please add a screenshot?

**Comment by hooloovooo** [comment]
I updated the component to use a simpler API. The styling is not entirely done yet, but some feedback on how the API looks is welcome.

**Comment by hooloovooo** [comment]
Added a option to ignore vars from destructed objects is the rest property is used as a fix to a lint issue i was having. Keep it or should i fix it some other way?

File: packages/core/src/layout/TabbedCard/TabbedCard.test.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+const minProps = {
+  title: 'Some title',
+  deepLink: {
+    title: 'A deepLink title',
+    link: '/mocked',
+  },
+};
+
+describe('<InfoCard />', () => {
```
Comment (stefanalund) [inline packages/core/src/layout/TabbedCard/TabbedCard.test.tsx:30]: TabbedCard?

**Review by nikek — DISMISSED** [review]

**Review by stefanalund — APPROVED** [review]

**Pull request merged**